src/Sylius/Bundle/WebBundle/Resources/views/Backend/Customer/show.html.twig

```diff
@@ -19,10 +19,8 @@
         {{ buttons.manage(path('sylius_backend_customer_index'), 'sylius.customer.manage'|trans) }}
         {% if not customer.deleted %}
             {{ buttons.edit(path('sylius_backend_customer_update', {'id': customer.id})) }}
-            {% if customer.user is null %}
-                {{ buttons.delete(path('sylius_backend_customer_delete', {'id': customer.id}), null, false, false) }}
-            {% elseif customer.user.id is not sameas(app.user.id) %}
-                {{ buttons.delete(path('sylius_backend_customer_delete', {'id': customer.id}), null, false, false) }}
+            {% if customer.user is null or customer.user.id is not sameas(app.user.id) %}
```

Comment:
Same as below, just copied it, I didn't want to change more things that needed in this PR.